changelog/2022-08-11T09_57_03+02_00_outputVerifier_takes_additional_clock

```diff
@@ -0,0 +1 @@
+CHANGE: `Clash.Explicit.Testbench.outputVerifier` now takes an additional clock as an argument: the clock used by the circuit under test. If your tests use the same domain for the test circuit and design under test, consider using `Clash.Explicit.Testbench.outputVerifier'`.
```

Comment:
Since `veryUnsafeSynchronizer` is part of the rather obscure but still _public_ API, could you add an entry for that as well?

Reply:
Good point and yes I can